src/ark/utils/plot_utils.py

```diff
@@ -740,15 +740,16 @@ def create_mantis_dir(fovs: List[str], mantis_project_path: Union[str, pathlib.P
     if not new_mask_suffix:
         new_mask_suffix = mask_suffix
 
-    map_df = map_df.loc[:, [f'{cluster_type}_meta_cluster', f'{cluster_type}_meta_cluster_rename']]
+    cluster_id_key = 'cluster_id' if cluster_type == 'cell' else f'{cluster_type}_meta_cluster'
```

Comment:
It's not a big deal at all, but `cluster_type` can only be `pixel` or `cell`. So I think it'd be a tad more clear if this was:
`cluster_id_key = 'cluster_id' if cluster_type == 'cell' else 'pixel_meta_cluster`